packages/plugins/plugin-markdown/src/components/MarkdownEditor.tsx

```diff
@@ -96,19 +96,18 @@ export const MarkdownEditor = ({
     onCommentSelect?.(id);
   });
 
-  // Focus comment.
+  // Focus the space that references the comment.
   useIntentResolver(MARKDOWN_PLUGIN, ({ action, data }) => {
     switch (action) {
+      // TODO(burdon): Use fully qualified ids everywhere.
```

Comment:
I know you just found a couple places where it wasn't being used but generally this is already the case and occasionally we find stragglers that we've missed causing issues.